src/pages/mainPage.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import SideNavbar from 'components/SideNavbar';
+import styled from 'styled-components';
+import { removeCookie } from 'components/Shared/Cookies';
+import { useNavigate } from 'react-router';
+
+const LoginBtn = styled.button`
+  background-color: black;
+  display: flex;
+  position: absoulte;
+  justify-content: center;
+  color: white;
+  width: 80px;
+  height: 32px;
+  margin-left: 1700px;
+  margin-top: -900px;
+  padding-top: 5px;
+`;
+
+const mainPage = () => {
+  const isLogined = useSelector((state) => state.auth.isLogined);
+  const navigate = useNavigate();
+  const loginToken = useSelector((state) => state.auth.token);
+  const [isTrued, setIsTrued] = useState(isLogined);
+  const loginClick = () => {
+    navigate(`/auth`);
+  };
+  useEffect(() => {
+    if (isLogined) {
+      setIsTrued((isTrued) => true);
+    } else {
+      setIsTrued((isTrued) => false);
+    }
+  }, [isLogined]);
```

Comment:
사용하지도 않는 `isTrued` 는 인자로 왜 받는 걸까요...? 더불어 의미없는 로직은 삼가주시길 바랍니당

```suggestion
  useEffect(() => setIsTrued(isLogined)}, [isLogined]);
```

Reply:
db5c7ad